CMakeLists.txt

```diff
@@ -217,6 +202,22 @@ else()
     message(STATUS "    LIBHWLOC_LIBRARY_DIRS = ${LIBHWLOC_LIBRARY_DIRS}")
 endif()
 
+if(hwloc_targ_SOURCE_DIR)
+    # apply security patch for HWLOC
+    execute_process(
+        COMMAND git apply ${PROJECT_SOURCE_DIR}/cmake/fix_coverity_issues.patch
+        WORKING_DIRECTORY ${hwloc_targ_SOURCE_DIR}
+        OUTPUT_VARIABLE UMF_HWLOC_PATCH_OUTPUT
+        ERROR_VARIABLE UMF_HWLOC_PATCH_ERROR)
+
+    if(UMF_HWLOC_PATCH_OUTPUT)
+        message(STATUS "HWLOC patch command output:\n${UMF_HWLOC_PATCH_OUTPUT}")
+    endif()
+    if(UMF_HWLOC_PATCH_ERROR)
+        message(WARNING "HWLOC patch command output:\n${UMF_HWLOC_PATCH_ERROR}")
```

Comment:
This output looks like something went wrong, although the patch does not apply because it has been already applied:
```
CMake Warning at CMakeLists.txt:217 (message):
  HWLOC patch:

  error: patch failed: hwloc/topology-x86.c:1375

  error: hwloc/topology-x86.c: patch does not apply
```
The build continues, as expected, though.

Reply:
Perhaps add `--allow-empty` to the `git apply` command?